Assets/MRTK/SDK/Features/Utilities/Solvers/TapToPlace.cs

```diff
@@ -167,6 +202,12 @@ public UnityEvent OnPlacingStopped
 
         protected internal bool IsColliderPresent => gameObject != null ? gameObject.GetComponent<Collider>() != null : false;
 
+        /// <summary>
+        /// The default value for SurfaceNormalOffset if UseDefaultSurfaceNormalOffset is true.  This value ensures an object
+        /// will be placed in alignment with a surface.
+        /// </summary>
+        private float defaultSurfaceNormalOffset => gameObject.GetComponent<Collider>().bounds.extents.z;
+
```

Comment:
Considered caching this property but decided not to in case the user changes the size of the object while they are placing it.

Reply:
Nit: This would make for a helpful comment (i.e. not caching explicitly to support XYZ). Otherwise it would be easy to "optimize" this way and lose support for that scenario.